osu.Game.Rulesets.Taiko/Skinning/LegacyDrumRoll.cs

```diff
@@ -76,9 +76,9 @@ public Color4 AccentColour
 
         private void updateAccentColour()
         {
-            headCircle.AccentColour = accentColour;
-            body.Colour = accentColour;
-            end.Colour = accentColour;
+            headCircle.AccentColour = LegacyColourCompatibility.DisallowZeroAlpha(accentColour);
+            body.Colour = LegacyColourCompatibility.DisallowZeroAlpha(accentColour);
+            end.Colour = LegacyColourCompatibility.DisallowZeroAlpha(accentColour);
```

Comment:
This is a nit, but you could call once and store to a local instead of calling thrice. (Unsure if this will be optimised anyway, but it'd read a little nicer for sure.)